deploy/cluster-libchart/templates/_rbac.tpl

```diff
@@ -0,0 +1,73 @@
+{{/*
+Define the service account name
+*/}}
+{{- define "cluster-libchart.serviceAccountName" -}}
+{{- printf "kb-sa-%s" (include "cluster-libchart.clusterName" .) }}
+{{- end }}
+
+{{/*
+Define the role name
+*/}}
+{{- define "cluster-libchart.roleName" -}}
+{{- printf "kb-role-%s" (include "cluster-libchart.clusterName" .) }}
+{{- end }}
+
+{{/*
+Define the rolebinding name
+*/}}
+{{- define "cluster-libchart.roleBindingName" -}}
+{{- printf "kb-rolebinding-%s" (include "cluster-libchart.clusterName" .) }}
+{{- end }}
+
+{{/*
```

Comment:
perhaps add a single function that include all of the following resources so the cluster chart only need a single "include" or "template"

Reply:
done.